pkg/ring/ring.proto

```diff
@@ -9,7 +9,9 @@ option (gogoproto.unmarshaler_all) = true;
 
 message Desc {
 	map<string,IngesterDesc> ingesters = 1 [(gogoproto.nullable) = false];
-	repeated TokenDesc tokens = 2 [(gogoproto.nullable) = false];
+
+	// Denormalised tokens used field number 2. We don't reserve this field, since we don't expect to run a mix
```

Comment:
IMO, If we aren't going to reserve the field, there is no reason for the comment.

Reply:
Right. I wanted to add that comment somewhere, but wasn't sure where. I've now also removed previously reserved fields, for the same reason.

Reply:
I think the best place to note it would be in the changelog